packaging/test_irods_client_rest_cpp.py

```diff
@@ -350,6 +350,35 @@ def test_list_with_accoutrements(self):
                 os.remove(file_name)
                 admin.assert_icommand(['irm', '-f', file_name])
 
+    def test_query_handles_case_insensitivity__issue124():
+        with session.make_session_for_existing_admin() as admin:
+            try:
+                file_count = 100
+                case_insensitive = 0
+
+                dir_name = 'test_query_collection_directory'
+                lib.make_large_local_tmp_dir(dir_name, file_count, 1024)
+
+                token  = irods_rest.authenticate('rods', 'rods', 'native')
+
+                query_lower = "SELECT DATA_PATH, COLL_NAME, DATA_NAME, DATA_SIZE WHERE DATA_NAME LIKE '\%i%'"
+                query_upper = "SELECT DATA_PATH, COLL_NAME, DATA_NAME, DATA_SIZE WHERE DATA_NAME LIKE '%I%'"
```

Comment:
why the difference in backslash on 364 and 365?

Reply:
My syntax highlighter was telling me that the lowercase 'i' was trying to be parsed as a format specifier. Some quick fiddling has shown that Python doesn't actually care.

Reply:
ah, very good.